_includes/nav-bar.html

```diff
@@ -55,6 +55,7 @@
                 Contribute <span class="caret"></span></a>
           <ul class="dropdown-menu">
             <li><a href="/documentation/contributing">Contribute</a></li>
+            <li><a href="/documentation/contributing/pullrequests.html">Handling Contributions</a></li>
```

Comment:
I prefer "Handling Contributions" to stick to the terminology of the navigation menu.
If someone else agrees, I can change it though.